test/csuite/timestamp_abort/main.c

```diff
@@ -374,9 +612,30 @@ thread_ckpt_run(void *arg)
     (void)unlink(ckpt_file);
     testutil_check(td->conn->open_session(td->conn, NULL, NULL, &session));
     first_ckpt = true;
+
+    /*
+     * Find the last successful backup.
+     */
+    if (use_backups && td->workload_iteration > 0) {
+        testutil_check(session->open_cursor(session, "backup:query_id", NULL, NULL, &cursor));
+        while ((ret = cursor->next(cursor)) == 0) {
+            testutil_check(cursor->get_key(cursor, &str));
+            testutil_assert(strncmp(str, "ID", 2) == 0);
+            u = (uint32_t)atoi(str + 2);
+            if (u > last_backup)
+                last_backup = u;
+        }
+        testutil_assert(ret == WT_NOTFOUND);
+        testutil_check(cursor->close(cursor));
```

Comment:
It may be more useful to have the backup thread separate from checkpoint. It is better coverage to have them active at the same time sometimes.